backend/src/api/v1/activities/services/weather_match_service.py

```
@@ -0,0 +1,346 @@
+"""
+날씨 매칭 점수 계산 서비스
+
+현재 날씨 조건과 각 장소/활동의 적합도를 0-100점으로 계산하여 제공합니다.
+"""
+
+from typing import Dict, List, Optional
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)